### DISCUSSION TITLE
S21 Faikin and Modbus RTD-RA

### DISCUSSION THREAD

**Discussion by FredericEn** [revk/xESP32-Faikout#145] [General]
Hey

Just bought some Daikin aircons, type Emura.
In small, there is written that when you use anything plugged in the S21 port to control the units with, the standard WiFi will not work. (Correct?)

I have a small PLC running at home and would like to use the Daikin RTD-RA with RS485.
But I don’t want to lose the ability of the Daikin app.

Is there a splitter available for the S21 port so I can combine a Faikin (so I don’t lose the app) and the RTD-RA?
Or is there a better way?

Thanks
Frederic

**Comment by revk** [comment]
I hear some will work with both but no idea if that is the case for your model. I would expect so if not unplugging an existing board...

Sorry I don't know about the RS485 interface.

I don't think there are S21 "splitters".

Someone else may know.
